flink/v1.15/flink/src/main/java/org/apache/iceberg/flink/source/StreamingMonitorFunction.java

```diff
@@ -139,7 +143,24 @@ public void run(SourceContext<FlinkInputSplit> ctx) throws Exception {
     }
   }
 
-  private void monitorAndForwardSplits() {
+  private long maxReachableSnapshotId(long lastConsumedSnapshotId, long latestSnapshotId, int maxSnapshotCount) {
```

Comment:
nit: maybe call the args `fromSnapshotIdExclusive` and `toSnapshotIdInclusive` to be consistent with recent incremental API naming

Reply:
actually maybe call this method as `toSnapshotIdInclusive(long lastConsumedSnapshotId, long currentSnapshotId, int maxPlanningSnapshotCount)`